pandas/io/pytables.py

```diff
@@ -779,6 +779,7 @@ def get(self, key: str):
         Parameters
         ----------
         key : str
+            Object to retrieve from file. Raises KeyError if not found.
```

Comment:
Future note: numpydoc supports a `Raises` section where you can detail the `KeyError`